config/application.properties

```diff
@@ -39,6 +48,9 @@ retro-game.noob-protection-multiplier=5
 # Inactivity
 retro-game.short-inactive-number-of-days=7
 retro-game.long-inactive-number-of-days=24
+# Password reset
+retro-game.password-reset-mail-from=no-reply@demo.intinte.org
```

Comment:
I think we shouldn't point to any specific website in the config, can we make the domain `localhost` or `retro-game.local`? Also, same above in `retro-game.domain`.

Reply:
Done